Sync/RequestExtensions.swift

```diff
@@ -7,18 +7,48 @@ import Alamofire
 import Shared
 
 extension Request {
-    public func responseParsedJSON(completionHandler: ResponseHandler) -> Self {
-        return response(serializer: Request.ParsedJSONResponseSerializer(), completionHandler: { (request, response, JSON, error) in
+    public func responseParsedJSON(partial: Bool, completionHandler: ResponseHandler) -> Self {
```

Comment:
We'll be using these request extensions elsewhere, so I'd like for us to make the choice deliberately each time. And I'd also like for us to tighten up for Sync itself, too — I just didn't want to thread params around at this stage in the game.